pyrk/neutronics.py

```diff
@@ -6,6 +6,7 @@
 from data import decay_heat as dh
 from reactivity_insertion import ReactivityInsertion
 from timer import Timer
+from th_component import THSuperComponent
```

Comment:
I don't think I want neutronics to depend on TH. Let's map out the dependency tree at some point.